rosdep/python.yaml

```diff
@@ -373,13 +373,31 @@ python-clearsilver:
     saucy: [python-clearsilver]
     trusty: [python-clearsilver]
 python-click:
+  debian:
+    sid: [python-click]
```

Comment:
We prefer not to define sid rules since sid's a rolling release they end up out of date very quickly.